solrcloudpy/__init__.py

```diff
@@ -4,5 +4,5 @@
 import logging
 logging.basicConfig()
 
-__version__ = "1.7.460"
+__version__ = "1.8.0"
```

Comment:
A note about the version number that's not obvious: `1.7` is solrcloudpy's version, while `460` tracks the solr version it's know to be compatible with, in this case `4.6.0`. You might decide to not stick with that scheme, I just wanted to make you aware of it